src/test/java/MyTest.java

```diff
@@ -0,0 +1,27 @@
+package src.test.java;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221401266-7d0e9187-e35d-4f6f-9f33-0fc80bafe59d.png)


![image](https://user-images.githubusercontent.com/104761482/221401247-9a08e534-ffea-4176-bb9b-dbe2acf06254.png)